spec.html

```diff
@@ -142,13 +141,13 @@ <h1>InsertUnicodeExtension( _locale_, _extension_ )</h1>
         The InsertUnicodeExtension abstract operation inserts _extension_, which must be a Unicode locale extension sequence, into _locale_, which must be a String value with a structurally valid and canonicalized BCP 47 language tag. The following steps are taken:
       </p>
       <p>
-        The following algorithm refers to <a href="https://tools.ietf.org/html/rfc5646#section-2.1">RFC 5646's Language-Tag grammar</a>.
+        The following algorithm refers to <a href="http://www.unicode.org/reports/tr35/#Identifiers">UTS 35's Unicode Language and Locale Identifiers grammar</a>.
```

Comment:
You missed https here and in a few other places below

Reply:
opps! sorry